staff/carolina-zhou/array-functions/reverse.test.js

```diff
@@ -0,0 +1,19 @@
+suite("reverse", function(){
+    test("reverse the direction of the elements, GOOD implementation", function(){
+        let array = [1, 2, 3, 4, 5]
```

Comment:
Dont use ES6 variable Declaration, use just ES5 Var declaration, and dont forget semicolons